dask/dataframe/io/parquet/arrow.py

```diff
@@ -2055,6 +2091,10 @@ def aggregate_metadata(cls, meta_list, fs, out_path):
         else:
             return meta
 
+    @classmethod
+    def multi_support(cls):
+        return cls == ArrowLegacyEngine
```

Comment:
Why not fastparquet?

Reply:
I did not enable multi-file reading for fastparquet yet in this PR. However, this logic is less about excluding other engines, and more about avoiding a change in behavior for engines that inherit from `ArrowLegacyEngine` (unless they explicitly opt in)

Reply:
Am I wrong in thinking that the speedup from the top of the PR would only apply to this engine, then?

Reply:
It is also enabled for `ArrowDatasetEngine` - So both Arrow-based engines will benefit.  However, I can add support for fastparquet as well today.